pkg/operator/handlers/dapr_handler.go

```diff
@@ -264,37 +278,32 @@ func (h *DaprHandler) createDaprServiceValues(ctx context.Context, expectedServi
 }
 
 func (h *DaprHandler) getAppID(wrapper ObjectWrapper) string {
-	annotations := wrapper.GetTemplateAnnotations()
-	if val, ok := annotations[appIDAnnotationKey]; ok && val != "" {
-		return val
-	}
-	return ""
+	annotationsMap := wrapper.GetTemplateAnnotations()
+	return annotationsMap[annotations.KeyAppID]
 }
 
 func (h *DaprHandler) isAnnotatedForDapr(wrapper ObjectWrapper) bool {
-	annotations := wrapper.GetTemplateAnnotations()
-	enabled, ok := annotations[daprEnabledAnnotationKey]
-	if !ok {
+	annotationsMap := wrapper.GetTemplateAnnotations()
+	val := annotationsMap[annotations.KeyEnabled]
+	if val == "" {
```

Comment:
why was this changed to not use ok?

Reply:
Good question.

First, context: `val, ok := annotationsMap[annotations.KeyEnabled]`:

- If the key does not exist on the map, `val == ""`
- You can have `val == ""` also if the key exists, but it's empty. That's where the `ok` is useful: in this case, `ok == true`

Now let's take a look at the behavior of the previous code vs current:

**Previous**
```go
enabled, ok := annotations[daprEnabledAnnotationKey]
if !ok {
	return false
}
return utils.IsTruthy(enabled)
```

**Current**

```go
val := annotationsMap[annotations.KeyEnabled]
if val == "" {
	return false
}
return utils.IsTruthy(val)
```

As you can see, in both cases the if the value doesn't exist, the returned value is `val == ""` so the function returns false. The additional check for `ok` was redundant.